third_party/rust/aead/v0_5/README.chromium

```diff
@@ -0,0 +1,7 @@
+Name: aead
+URL: https://crates.io/crates/aead
+Description: Traits for Authenticated Encryption with Associated Data (AEAD) algorithms, such as AES-GCM as ChaCha20Poly1305, which provide a high-level API
+Version: 0.5
+Security Critical: no
```

Comment:
Is this code not involved in zcash transactions, and so Security Critical in that it's processing untrusted input from the network?

Reply:
This is need to be reviewed, maybe @kdenhartog could help marking these packages properly and help decide whether crypto packages should be marked as security critical

Reply:
Usually crypto is security critical. The [main criteria](https://chromium.googlesource.com/chromium/src/+/main/docs/adding_to_third_party.md#Add-a-README_chromium) are handling complex data formats and handling untrusted input. Seems like both apply here?

Reply:
Looking into this now and will report back once I have a better idea about the best way to handle this. I believe marking these as security critical would be useful.

Reply:
@kylehickinson is this the only crate should be marked as security critical?